packages/jest-runner/package.json

```diff
@@ -13,6 +13,7 @@
     "@jest/console": "^24.7.1",
     "@jest/environment": "^24.8.0",
     "@jest/test-result": "^24.8.0",
+    "@jest/transform": "^24.8.0",
```

Comment:
needs to be added to https://github.com/facebook/jest/blob/3ab2fc1b0ae3542bb8cdc927bf5459ba16a6420c/packages/jest-runner/tsconfig.json#L7-L21

Reply:
done